### PR TITLE
Navigation setup

### PR DESCRIPTION
**Pull request by doersweb**
Changes:
1. Navigation setup
2. Package refactoring
3. Exception handling based on review comments
4. Added colors

### CODE REVIEW

File: app/src/main/java/com/doersweb/tasky/planner/data/remote/response/PlannerData.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.doersweb.tasky.planner.data.remote.response
+
+import com.doersweb.tasky.planner.data.util.PlannerType
+
+data class PlannerData(
+    val plannerType: PlannerType,
+    val otherData: String
```
Comment (philipplackner) [inline app/src/main/java/com/doersweb/tasky/planner/data/remote/response/PlannerData.kt:7]: What is otherData?

File: buildSrc/src/main/java/Retrofit.kt [hunk 2]
```diff
@@ -2,6 +2,7 @@ object Retrofit {
     private const val version = "2.9.0"
     const val retrofit = "com.squareup.retrofit2:retrofit:$version"
     const val moshiConverter = "com.squareup.retrofit2:converter-moshi:$version"
+    const val gsonConverter = "com.squareup.retrofit2:converter-gson"
```
Comment (philipplackner) [inline buildSrc/src/main/java/Retrofit.kt]: Why Gson in addition to Moshi?

**Review by philipplackner — APPROVED** [review]

File: app/src/main/java/com/doersweb/tasky/planner/data/remote/response/PlannerData.kt (hunk 1 above)
Comment (doersweb) [reply app/src/main/java/com/doersweb/tasky/planner/data/remote/response/PlannerData.kt:7]: This is just for representation purposes till I create the actual schema.

File: buildSrc/src/main/java/Retrofit.kt (hunk 2 above)
Comment (doersweb) [reply buildSrc/src/main/java/Retrofit.kt]: Actually no need, was trying something that dint work but missed removing it. Thanks

**Pull request merged**